modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/internal/model/listener/test/UserModelListenerTest.java

```diff
@@ -91,6 +105,36 @@ public void tearDown() throws Exception {
 		_companyLocalService.deleteCompany(_company.getCompanyId());
 	}
 
+	@Test
+	public void testOnAfterRemove() throws Exception {
+		BundleActivator bundleActivator =
+			new UserLocalServiceTestBundleActivator();
+
+		bundleActivator.start(_bundleContext);
+
+		User defaultServiceAccountUser = UserTestUtil.addUser(
+			"default-service-account");
```

Comment:
Hey @thallescezar could you please check if the `defaultServiceAccountUser` is already created, if so, please remove this statement and forward this pull request, otherwise all looks good to me!

Thank you!

Reply:
DONE